app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC
+
+from app.players.player import Player
+
+
+class Dwarf(Player, ABC):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str) -> None:
```

Comment:
Place each argument including `self` on a new line so that there would be no arguments on the same line with opening and closing parenthesis. It will help to improve the code style and readability. Apply this change to all corresponding classes